### PR TITLE
NF-229 - enter importer's EORI number

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -165,6 +165,14 @@ itemNumbers.hint = Tell us the item numbers as they appear on your declaration.
 itemNumbers.error.required = Enter an item number
 itemNumbers.error.length = Item numbers must be 500 characters or fewer
 
+importer.eori.title = What is the importer’s EORI number?
+importer.eori.hint = The number starts with GB and is followed by 12 digits. For example, GB345834921000.
```
Comment (crosslandwa) [inline conf/messages:169]: OBSERVATION: The content here is not consistent with the validation (which allows 12 _or_ 15 digits) - is this intentional?

Comment (SteveSugden) [reply conf/messages:169]: Good spot.  I'll ask.

File: app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/connectors/NIDACConnector.scala [hunk 2]
```diff
@@ -17,6 +17,7 @@
 package uk.gov.hmrc.nationalimportdutyadjustmentcentrefrontend.connectors
 
 import javax.inject.Inject
+import play.api.libs.json.{Json, OFormat}
```
Comment (rorticus0502) [inline app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/connectors/NIDACConnector.scala:20]: This import appears to be unused

File: app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/forms/mappings/Validation.scala [hunk 3]
```diff
@@ -23,6 +23,7 @@ object Validation {
   val safeInputPattern     = """^[A-Za-z0-9À-ÿ \!\)\(.,_/’'"&-]+$"""
   val entryProcessingUnit  = "^[0-9]{3}$"
   val entryNumber          = "^([0-9]{6}[a-z|A-Z])$"
+  val eoriNumber           = "^([G]{1}[B]{1}[0-9]{15})|([G]{1}[B]{1}[0-9]{12})$"
```
Comment (rorticus0502) [inline app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/forms/mappings/Validation.scala:26]: could be simplified slightly
^GB([0-9]{15}|[0-9]{12})$

File: test/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/views/makeclaim/CheckYourAnswersViewSpec.scala [hunk 4]
```diff
@@ -26,26 +26,26 @@ class CheckYourAnswersViewSpec extends UnitViewSpec with TestData {
 
   private val page = instanceOf[CheckYourAnswersView]
 
-  private val claim          = Claim(completeAnswers)
-  private val view: Document = page(claim, navigatorBack)
+  private val complateClaim                                = Claim(completeAnswers)
```
Comment (rorticus0502) [inline test/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/views/makeclaim/CheckYourAnswersViewSpec.scala:29]: typo

File: app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/forms/mappings/Validation.scala (hunk 3 above)
Comment (SteveSugden) [reply app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/forms/mappings/Validation.scala:26]: I know. I just copied the regex from PEGA schema.

**Review by rorticus0502 — APPROVED** [review]

File: app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/forms/mappings/Validation.scala (hunk 3 above)
Comment (rorticus0502) [reply app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/forms/mappings/Validation.scala:26]: Makes sense. If they provide one we should use it to save any arguments. But theirs could be improved.

**Pull request merged**